configobject/history/history.go

```diff
@@ -481,18 +496,26 @@ func acknowledgementHistoryWorker(super *supervisor.Supervisor) {
 			return data
 		},
 		func(values map[string]interface{}) []interface{} {
-			id := uuid.MustParse(values["id"].(string))
+			eventId := uuid.MustParse(values["event_id"].(string))
```

Comment:
Don't you want to inline this one like id?

Reply:
I would like to, but you can't.